src/main/webapp/assets/svg/forkandknife.svg

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="795.000000pt" height="795.000000pt" viewBox="0 0 795.000000 795.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.16, written by Peter Selinger 2001-2019
+</metadata>
+<g transform="translate(0.000000,795.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
```

Comment:
Would it be possible to change the fill to be something more like `#303030` (more dark grey than `#000000`?) The black's contrast looks a little harsh since we don't use pure black on any other big elements :)

Reply:
actually I don't think `var(--dark-grey)` will work because this isn't a CSS file with access to the variable declarations, but you should be able to use the same color mapped to the variable

Reply:
Yes, great idea! I changed it to a lighter grey.